.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+name: "Tests"
+
+on: [push, pull_request]
+
+jobs:
+  phpunit:
+    name: "PHPUnit tests"
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: ["7.1", "8.0"]
+        include:
+          - php: "7.1"
+            composer-args: "--prefer-lowest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php }}"
+          coverage: "none"
+
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: "~/.composer/cache"
+          key: ${{ matrix.php }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ matrix.php }}-composer-
+
+      - name: Cache phpunit dependencies
+        uses: actions/cache@v2
+        with:
+          path: "./vendor/bin/.phpunit"
+          key: ${{ matrix.php }}-phpunit-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ matrix.php }}-phpunit-
+
+      - name: Install dependencies
+        run: |
+          echo "::group::composer update"
+          composer update --no-interaction --no-progress ${{ matrix.composer-args }}
+          echo "::endgroup::"
+          echo "::group::install phpunit"
+          ./vendor/bin/simple-phpunit install
```

Comment:
Not sure if this line does anything.

This is the result I'm getting in my dev:

```
./vendor/bin/simple-phpunit -v install; echo $?"
PHPUnit 9.4.4 by Sebastian Bergmann and contributors.

Cannot open file "install".
0
```

Reply:
PHPUnit will installed lazily whenever this script is invoked. For CI purposes, it's preferable to do this during a setup stage instead of the actual test stage.

You'd have to parse through [`simple-phpunit.php`](https://github.com/symfony/phpunit-bridge/blob/15cab721487b7bf43ad545a1e7d0095782e26f8c/bin/simple-phpunit.php#L420) to make sense of this, but "install" is specially ignored by the script so that you can invoke this once without running any tests. It cannot be mixed with other PHPUnit command line args/options, such as `-v`.